justega247/favorite-things
# #167193399: Create category endpoint

#### What does this PR do?
- This PR handles the creation of category for the favorite things
#### Description of Task to be completed?
- create the endpoint that handles the creation of category
- write tests to confirm the functionality of the code
#### How should this be manually tested?
- Navigate to the root directory of your application
- Start the application using **python manage.py runserver**
- Navigate to the endpoint for creating a category http://127.0.0.1:8000/api/v1/category/ after you have supplied the necessary authentication
- On providing valid details, you should see the category created
#### What are the relevant pivotal tracker stories?
[#167193399](https://www.pivotaltracker.com/story/show/167193399)
